tests/test_core_sdk.py

```diff
@@ -1465,3 +1467,69 @@ def test_fetch_sessions():
     response = langfuse.fetch_sessions(limit=1, page=2)
     assert len(response.data) == 1
     assert response.data[0].id in [session1, session2, session3]
+
+
+def test_fetch_scores():
+    langfuse = Langfuse()
+
+    # Create a trace with a score
+    name = create_uuid()
+    trace = langfuse
+    trace = langfuse.trace(name=name)
+    trace.score(name="harmfulness", value=0.5)
+    trace.score(name="quality", value=1)
+    trace.score(name="relevance", value=0.8)
+    langfuse.flush()
+
+    # Fetch scores
+    response = langfuse.fetch_scores()
+
+    # Assert the structure of the response
+    assert isinstance(response, FetchScoresResponse)
+    assert hasattr(response, "data")
+    assert hasattr(response, "meta")
+    assert isinstance(response.data, list)
+    assert response.data[2].name == "harmfulness"
+    assert response.data[2].value == 0.5
+    assert response.data[1].name == "quality"
+    assert response.data[1].value == 1
+    assert response.data[0].name == "relevance"
+    assert response.data[0].value == 0.8
+
+    # fetch only one
+    response = langfuse.fetch_scores(limit=1, page=2)
+    assert len(response.data) == 1
+
+
+def test_fetch_prompts():
+    langfuse = Langfuse()
+
+    # Create a prompt
+    langfuse.create_prompt(
+        name="event-planner",
+        prompt="Plan an event titled {{Event Name}}. The event will be about: {{Event Description}}. "
+        "The event will be held in {{Location}} on {{Date}}. "
+        "Consider the following factors: audience, budget, venue, catering options, and entertainment. "
+        "Provide a detailed plan including potential vendors and logistics.",
+        config={
+            "model": "gpt-3.5-turbo-1106",
+            "temperature": 0,
+        },
+        labels=["production"],
+    )
+    langfuse.flush()
+
+    # Fetch prompts
+    response = langfuse.fetch_prompts()
+
+    # Assert the structure of the response
+    assert isinstance(response, FetchPromptsResponse)
+    assert hasattr(response, "data")
+    assert hasattr(response, "meta")
+    assert isinstance(response.data, list)
+    assert response.data[0].name == "event-planner"
+    assert (
+        response.data[0].prompt
```

Comment:
why do we have the prompt here included? the api does not include the prompt in the response: https://api.reference.langfuse.com/#get-/api/public/v2/prompts